pandas/tests/scalar/timedelta/test_constructors.py

```diff
@@ -241,7 +248,6 @@ def test_iso_constructor(fmt, exp):
         "PDTHMS",
         "P0DT999H999M999S",
         "P1DT0H0M0.0000000000000S",
-        "P1DT0H0M00000000000S",
```

Comment:
is this not legit?

Reply:
This test tests expected failures, and this *is* rather a legitimate duration.  It will succeed with the current code, and I moved this now to the tests where the parsing should succeed.

Reply:
ahh kk great